activemq-broker/README.md

```diff
@@ -0,0 +1,6 @@
+## Description
```

Comment:
I'm not a big fan of having `README.md` in a module. It can lead to scattered the information in different place.
I would rather update the `README.md` at root level. We can eventually add a `CONTRIBUTING.md` file dedicate to contribution.

Reply:
Ok sounds good. Will remove it at the module level. Because this is work in progress so I don't want to increase its visibility. But I will push the final documentation to the root level README when it's mature and we can review that :)